test/forkMainnet/GenericSwap.t.sol

```diff
@@ -183,12 +185,21 @@ contract GenericSwapTest is Test, Tokens, BalanceUtil, Permit2Helper, SigHelper
 
         mockStrategy.setOutputAmountAndRecipient(gsData.makerTokenAmount, payable(address(genericSwap)));
         vm.expectEmit(true, true, true, true);
-        emit Swap(getGSDataHash(gsData), gsData.maker, taker, taker, gsData.takerToken, gsData.takerTokenAmount, gsData.makerToken, gsData.makerTokenAmount);
+        emit Swap(
+            getGSDataHash(gsData),
+            gsData.maker,
+            taker,
+            taker,
+            gsData.takerToken,
+            gsData.takerTokenAmount,
+            gsData.makerToken,
+            gsData.makerTokenAmount - 1
```

Comment:
I think we can have `expectedOut = gsData.makerTokenAmount - 1` for these assertion.

Reply:
Good idea, but maybe call it `realChangedInGS` for consistency.